.stickler.yml

```diff
@@ -0,0 +1,4 @@
+
+linters:
+  eslint:
+    config: './.eslintrc.json'
```

Comment:
Let's make sure `stickler` is ON for this repo.

- Goto https://stickler-ci.com/
- Login
- ON the linter for the repo

![image](https://user-images.githubusercontent.com/36057474/72263039-66cc4700-3618-11ea-8a4a-85382ef14e37.png)

Reply:
Bola, if you look at the screenshot I shared at the end of my PR comment, you'll notice that Stickler-CI was (and still is) enabled for this repository. Here is another screenshot taken 5 minutes ago:

![image](https://user-images.githubusercontent.com/22228397/72263503-fa851f80-35e6-11ea-8419-98de7d755447.png)

Stickler simply doesn't work on this repo, and I don't know why.

Reply:
Yes, Adriaan. Please see the difference between my screenshot and yours. Mine is ON and yours is OFF (hence the grey colour).

Also, if you ON it there would be no difference in the PR unless you push a commit to trigger it.

Reply:
Yes you are right. I thought they were ON. Thanks for pointing this out. Finally, it all makes sense.

Reply:
It works now, thanks!